plantuml-utilities.cmake

```diff
@@ -0,0 +1,93 @@
+# cmake-utilities
+#
+# Copyright 2019, 2020 Andrew Countryman <apcountryman@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# File: plantuml-utilities.cmake
+# Description: CMake PlantUML utilities.
+
+find_program( PLANTUML plantuml )
+mark_as_advanced( PLANTUML )
+if( PLANTUML STREQUAL "PLANTUML-NOTFOUND" )
+    message( FATAL_ERROR "plantuml not found" )
+endif( PLANTUML STREQUAL "PLANTUML-NOTFOUND" )
+
+# Generate a PlantUML diagram.
+#
+# SYNOPSIS
+#     add_plantuml_diagram(
+#         <source>
+#         [FILE <file>]
+#         [TARGET <target>]
+#     )
+#
+# OPTIONS
+#     <source>
+#         The PlantUML source file to generate the diagram from.
+#     FILE <file>
+#         The variable to store the absolute path of the generated diagram in.
+#     TARGET <target>
+#         The variable to store the name of the target that triggers generation of the
+#         diagram file in.
```

Comment:
Change wording to `The variable to store the name of the target that triggers generation of the diagram in`.